src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -502,57 +502,152 @@ private string JoinPackageNames(IEnumerable<Package> pkgs)
         /// Warn about duplicate package conflicts depending on whether the package is 
         /// a built-in package and whether versions match.
         /// </summary>
-        /// <param name="name">name of package being downloaded</param>
         /// <param name="package">package version being downloaded</param>
-        /// <param name="duplicatePackage">local package found to be duplicate of one being downloaded</param>
-        private void WarnAboutDuplicatePackageConflicts(string name, PackageVersion package, Package duplicatePackage)
-        {
-            if (duplicatePackage == null) return;
-
-            var packageToDownload = $"{name} {package.version}";
-            if (duplicatePackage.BuiltInPackage)
+        /// <param name="duplicatePackage">local package found to be duplicate of one being downloaded (not including the main package)</param>
```

Comment:
I don't understand this comment.

Reply:
The `not including main package` comment is confusing. I think the `duplicatepackage` refers to the local package that conflicts with the main package that is being downloaded, right?

Reply:
`(not including the main package)` was meant for the other input parameter `conflicts`
Will move it